builder/vmware/common/driver.go

```diff
@@ -23,16 +23,39 @@ import (
 )
 
 const (
+	// OVF Tool.
 	ovfToolDownloadURL = "https://developer.broadcom.com/tools/open-virtualization-format-ovf-tool/latest"
 	ovfToolMinVersion  = "4.6.0"
+
+	// Architectures.
+	archAMD64 = "x86_x64"
+	archARM64 = "arm64"
+
+	// Operating systems.
+	osWindows = "windows"
+
+	// Clone types.
+	cloneTypeLinked = "linked"
+	cloneTypeFull   = "full"
+
+	// GUI arguments.
+	guiArgumentNoGUI = "nogui"
+	guiArgumentGUI   = "gui"
+
+	// Application binary names.
+	appVdiskManager = "vmware-vdiskmanager"
+	appVmrun        = "vmrun"
+	appVmx          = "vmware-vmx"
+
+	// Version Regular Expressions.
+	productPreviewRegex = `(?i)VMware [a-z0-9-]+ e\.x\.p `
+	productVersionRegex = `(?i)VMware [a-z0-9-]+ (\d+\.\d+\.\d+)`
+	ovfToolVersionRegex = `\d+\.\d+\.\d+`
```

Comment:
I would suggest they all become global vars so the regexes are compiled directly, the raw regex string can remain a const if you prefer, but having them all as `regexp.MustCompile` will help if we make them evolve and they break later, as we'll be able to immediately correct them.

Reply:
Resolved in https://github.com/hashicorp/packer-plugin-vmware/commit/018e73a30fac537bffa232911015f53fdc8998e8. 😄